.gitlab-ci.yml

```diff
@@ -0,0 +1,13 @@
+# SPDX-FileCopyrightText: None
+# SPDX-License-Identifier: CC0-1.0
+
+variables:
+  GIT_SUBMODULE_STRATEGY: recursive
+
+include:
+  - project: sysadmin/ci-utilities
+    file:
+      - /gitlab-templates/linux.yml
+      - /gitlab-templates/freebsd.yml
+      - /gitlab-templates/linux-qt6.yml
+      - /gitlab-templates/freebsd-qt6.yml
```

Comment:
I assume these files are from a gitlab fork?
I don't mind them too much if they make future contributions easier, but if they were added by accident it would be nice to remove them.

Reply:
Yes. I'm testing to build the plugin on the KDE Ci so far they built but I have to add the dependencies to the ci image.
We can also use the KDE Ci testing if we investigate how to start the KDE Gitlab runner here.

Reply:
Cool, I wasn't aware that KDE has a public CI. That's probably worth investigating 👍